sidecar/proxy/envoy/manager.go

```diff
@@ -405,7 +404,7 @@ func buildRoutes(ruleList []api.Rule) []Route {
 
 				runtime := &Runtime{
 					Key:     buildWeightKey(backend.Name, backend.Tags),
-					Default: 0,
+					Default: int(backend.Weight * 100),
```

Comment:
by spec you mean the rules schema?  Just to verify this will mean re-writing all of the examples docs and CLI to use set integers vs floats when creating the rules